docs/userguide/packaging/plug-ins.md

```diff
@@ -33,66 +33,74 @@ uninstall themselves before they are removed from memory and unloaded.
 The current plugin API is documented as
 [icuplug.h](http://icu-project.org/apiref/icu4c434/icuplug_8h.html)
 Some sample plugins are available at:
-[testplug.c](http://bugs.icu-project.org/trac/browser/icu/trunk/source/tools/icuinfo/testplug.c)
+[testplug.c](../../../icu4c/source/tools/icuinfo/testplug.c)
 Here is a simple, trivial plugin:
 
-       U_CAPI
-     **UPlugTokenReturn** U_EXPORT2 **myPlugin** (UPlugData *data, UPlugReason reason, UErrorCode *status) {
-         if(reason==UPLUG_REASON_QUERY) {
-             uplug_setPlugName(data, "Simple Plugin"); /* optional */
-             uplug_setPlugLevel(data, UPLUG_LEVEL_HIGH); /* Mandatory */
-         } else if(reason==UPLUG_REASON_LOAD) {
-             /* ... load ... */
-             /* Set up some ICU things here. */
-         } else if(reason==UPLUG_REASON_UNLOAD) {
-             /* ... unload ... */
-         }
-         return UPLUG_TOKEN; /* Mandatory. */
-     }
-
-The UPlugData\* is an opaque pointer to the plugin-specific data, and is used in
+```c
+  U_CAPI
+**UPlugTokenReturn** U_EXPORT2 **myPlugin** (UPlugData *data, UPlugReason reason, UErrorCode *status) {
+    if(reason==UPLUG_REASON_QUERY) {
+        uplug_setPlugName(data, "Simple Plugin"); /* optional */
+        uplug_setPlugLevel(data, UPLUG_LEVEL_HIGH); /* Mandatory */
+    } else if(reason==UPLUG_REASON_LOAD) {
+        /* ... load ... */
+        /* Set up some ICU things here. */
+    } else if(reason==UPLUG_REASON_UNLOAD) {
+        /* ... unload ... */
+    }
+    return UPLUG_TOKEN; /* Mandatory. */
+}
+```
+
+The UPlugData* is an opaque pointer to the plugin-specific data, and is used in
 all other API calls.
 The API contract is:
+
 1. the plugin MUST always return UPLUG_TOKEN as a return value- to
 indicate that it is a valid plugin.
+
 2. when the 'reason' parameter is set to UPLUG_REASON_QUERY, the
```

Comment:
Suggest doing automatic numbering instead of manual numbering for numbered list.

Reply:
you mean 1. 1. 1. 1. ?  except this way it's more readable if viewed as plain text

Reply:
I mean 1. 1. 1. 1.

I know it is more readable in plain text. But the moment you have to change the order of things, it becomes more painful to renumber. Automatic numbering takes care of that.

Reply:
I am with Steven here -- I prefer manual numbering for reading the .md source.
I might change my mind when we have a very long list, or one that we keep changing a lot.

Reply:
I think a linter might be able to verify that the items are kept in order